### DISCUSSION TITLE
Prowler checks versus Security Hub or other AWS Services checks

### DISCUSSION THREAD

**Discussion by cerontrustly** [prowler-cloud/prowler#2417] [Q&A]
Hello guys!

I was wondering if you have a list of Prowler checks, or maybe documented somewhere, which are not covered by Security Hub or any other AWS Service. I mean those checks only Prowler does. In the past I know we had those extra checks...

Thank you,
Igor

**Comment by toniblyx** [comment]
Hi @cerontrustly, we don't have such a comparison but I have just created a spreadsheet with both list of checks and you can see and pair it manually if you want. It is not sorted though. If do do that comparison and share it back to us we can make it public later
[Prowler and SH checks list.xlsx](https://github.com/prowler-cloud/prowler/files/11597478/Prowler.and.SH.checks.list.xlsx)
